src/api/index.tsx

```diff
@@ -77,6 +79,20 @@ export const makeApiURL = ({
   return parts.join('');
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const makeQueryString = (query: { [key: string]: any }) => {
```

Comment:
How about this?

```diff
diff --git a/src/api/index.tsx b/src/api/index.tsx
index 5e66083..6a58351 100644
--- a/src/api/index.tsx
+++ b/src/api/index.tsx
@@ -79,8 +79,9 @@ export const makeApiURL = ({
   return parts.join('');
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const makeQueryString = (query: { [key: string]: any }) => {
+export const makeQueryString = (query: {
+  [key: string]: string | number | null | void | boolean;
+}) => {
   const resolvedQuery = { ...query };
   Object.keys(resolvedQuery).forEach((key) => {
     const value = resolvedQuery[key];
```

It would catch passing object values by mistake